src/js/themes/base.js

```diff
@@ -1523,10 +1523,10 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           },
         },
       },
-      // container: {
-      //   // any box props,
-      //   extend: undefined,
-      // },
+      container: {
```

Comment:
Do we need to uncomment this? can we leave it commented out like before

Reply:
done